lib/cmd-utils/get-contextual-tuples.go

```diff
@@ -0,0 +1,33 @@
+package cmdutils
+
+import (
+	"strings"
+
+	"github.com/openfga/cli/lib/clierrors"
+	"github.com/openfga/go-sdk/client"
+	"github.com/spf13/cobra"
+)
+
+func ParseContextualTuples(cmd *cobra.Command) ([]client.ClientTupleKey, error) {
+	contextualTuples := []client.ClientTupleKey{}
+	contextualTuplesArray, _ := cmd.Flags().GetStringArray("contextual-tuple")
+
+	if len(contextualTuplesArray) > 0 {
```

Comment:
FYI ideally in the next iteration of tests we add tests that mimic cli calling directly. aka cli input output.

Because there is still some logic we miss for our tests now unfortunately